plugins/octopus-deploy/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@backstage/plugin-octopus-deploy",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
```

Comment:
Oh this needs to be removed so it gets published correctly.

Reply:
Do I remove all of these lines? Or just the `"private": true,` line?

Reply:
Just the private!

Reply:
Okay, done :).